CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Change Log
 
+## [v1.13.2](https://github.com/auth0/auth0-spa-js/tree/v1.13.2) (2020-11-09)
+
+[Full Changelog](https://github.com/auth0/auth0-spa-js/compare/v1.13.1...v1.13.2)
+
+**Added**
+
+- [SDK-2121] Add support for token validation for Organizations [\#631](https://github.com/auth0/auth0-spa-js/pull/631) ([stevehobbsdev](https://github.com/stevehobbsdev))
```

Comment:
Shouldnt this be 1.14.0 ?

Reply:
As it's an internal non-breaking change without any public API changes, I figured a patch version would be more appropriate but happy to hear otherwise.

Reply:
Sure, I am fine with either. Just wanted to ensure this was intended.